client/src/service.rs

```diff
@@ -90,8 +90,8 @@ where
         self.machine.tick(local_time);
     }
 
-    fn wake(&mut self) {
```

Comment:
I'm in favor to keep wake because it is common in rust async, when a task need to be wakeup, and I think this is what the code does?

Reply:
The issue with wake here is that we also use `wake` to wakeup the reactor's event loop. Here it's waking up the state machine due to a timer. So I think this should be `timer_expired`